### PR TITLE
Bug 1803723: [Bug][AzureToolkit][Test] Error box begin with failed to load/refresh items of combo box

### PR DESCRIPTION
**Pull request by wangmingliang-ms**
… load/refresh items of combo box

### CODE REVIEW

File: Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/operation/AzureOperationRef.java [hunk 1]
```diff
@@ -23,15 +23,26 @@
...
     private Object instance;
+
+    @Override
+    public boolean equals(final Object obj) {
+        if (!(obj instanceof AzureOperationRef)) {
+            return false;
+        }
+        final AzureOperationRef operation = (AzureOperationRef) obj;
+        return operation.getMethod() == this.method || operation.getMethod().equals(this.method);
+    }
```
Comment (shenqianjin) [inline Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/operation/AzureOperationRef.java:47]: only "operation.getMethod().equals(this.method)" should be enough. right ?

File: Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/handler/AzureExceptionHandler.java [hunk 2]
```diff
@@ -53,11 +56,13 @@ public static void onRxException(final Throwable e) {
     }
 
     public void handleException(Throwable throwable, @Nullable AzureExceptionAction... action) {
+        log.log(Level.WARNING, "caught an error in AzureExceptionHandler", throwable);
```
Comment (shenqianjin) [inline Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/handler/AzureExceptionHandler.java:59]: I prefer to log.log(Level.WARNING, throwable);

File: Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/operation/AzureOperationRef.java (hunk 1 above)
Comment (wangmingliang-ms) [reply Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/operation/AzureOperationRef.java:47]: yes

File: Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/handler/AzureExceptionHandler.java (hunk 2 above)
Comment (wangmingliang-ms) [reply Utils/azuretools-core/src/com/microsoft/azure/toolkit/lib/common/handler/AzureExceptionHandler.java:59]: so do I, but there is no such method.

**Pull request merged**